src/extension.ts

```diff
@@ -78,6 +78,20 @@ export async function activate(context: vscode.ExtensionContext): Promise<vscode
 			outputChannel
 		});
 	});
+	const removeDiagnosticsOnActiveFile = vscode.commands.registerCommand(Constants.COMMAND_REMOVE_DIAGNOSTICS_ON_ACTIVE_FILE, (selection: vscode.Uri, multiSelect?: vscode.Uri[]) => {
```

Comment:
I understand that multi select is a think for the selected files from the file tree. But for the active file (which I'm assuming comes up with you right click on a file)... where does multiselect come into play?

I noticed on line 67, our COMMAND_RUN_ON_ACTIVE_FILE doesn't even use the selection or multiSelect arguments. This is making me wonder how that even works because it doesn't even take in the active file as a selection. We just pass in [] there. Hmm.

Reply:
targeting.getTargets() - line 42 gets the active editor. In order to reach that code, you pass in [] file here. All the file selection logic is encapsulated in that method (that adds some additional logic for Windows). May not be directly useful here, but still that seems to be the pattern I saw in other implementation and followed the same logic.